src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -708,7 +776,7 @@ Java_sun_awt_windows_WPageDialogPeer__1show(JNIEnv *env, jobject peer)
         AwtPrintControl::setPrintHDName(env, self, setup.hDevNames);
     }
 
-done:
+
```

Comment:
```suggestion
```

Reply:
To be clear: I recommend deleting this line, since it does not make sense to have two consecutive blank lines here.